src/components/ModerationBar/index.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import { createStore } from 'redux';
+
+import { Store, reducer } from '../../state';
+import { Styling } from '../../utils/storybook';
+import APIClient from '../../api';
+import { setErrors, updateModerationState } from '../../actions/moderation';
+
+import ModerationBar from './index';
+import { ModerationErrorCode } from '../../state/moderation';
+
+export default { title: 'ModerationBar' };
+
+function RenderModerationBarForStorybok({ store }: { store: Store }) {
```

Comment:
I do quite like the idea of using `Storybok`, but I think tbh we should maybe go with the more established `Storybook`.

Reply:
But `Storybok` is so much better! We save one byte every time we use it, which adds up you know...